stl/inc/xutility

```diff
@@ -3469,13 +3488,17 @@ _NODISCARD _CONSTEXPR17 bool operator==(const move_iterator<_Iter1>& _Left, cons
 
 #if !_HAS_CXX20
 template <class _Iter1, class _Iter2>
-_NODISCARD _CONSTEXPR17 bool operator!=(const move_iterator<_Iter1>& _Left, const move_iterator<_Iter2>& _Right) {
+_NODISCARD _CONSTEXPR17 bool
+    operator!=(const move_iterator<_Iter1>& _Left, const move_iterator<_Iter2>& _Right) noexcept(
+        noexcept(_Fake_copy_init<bool>(!(_Left == _Right)))) /* strengthened */ {

```

Comment:
`_Left == _Right` is a `bool`. We know that `!` won't throw, and neither will conversion from `bool` to `bool`:
```suggestion
        noexcept(_Left == _Right)) /* strengthened */ {
```